### DISCUSSION TITLE
Configuring SEB as DaaS

### DISCUSSION THREAD

**Discussion by andrewd1970** [SafeExamBrowser/seb-win-refactoring#837] [Q&A]
Our IT is moving SEB from local installations to part of the DaaS. We are having problems configuring it to work in our Moodle. has anyone deployed SEB as DaaS and can you suggest how to set up the configuration?

**Comment by danschlet** [comment]
Are you aware that the main purpose of SEB is to secure physical machines, so no other applications can be used? How do you want to achieve that if SEB sorely runs in a virtual desktop?

We do sometimes use SEB instances with virtual desktops, but only together another SEB instance which secures the physical machine. See [this paper](https://safeexambrowser.org/vdi/Competence-Oriented_Exams_VDI+SEB_ETH-LET_2023_05.pdf).

**Reply by andrewd1970** [reply]
Thanks - I'll pass this along to our IT
